NICE.Identity.Management/ClientApp/src/components/UnlockUser/UnlockUser.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react";
+
+import { Endpoints } from "../../data/endpoints";
+
+type UnlockUserProps = {
+	id: number;
+	isBlocked: boolean;
+	onToggleLock: Function;
+	onError: Function;
```

Comment:
I think it would be better to type these functions with their proper arguments. For example onError is probably:

```suggestion
	onError: (error: Error) => void;
```